src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/CertCollectionLoader.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading;
+
+namespace Internal.Cryptography.Pal
+{
+    internal class CertCollectionLoader : ILoaderPal
```

Comment:
Sealed all the new types; then made a new commit to seal out the old types that could be.

The question really is "when there's no InternalsVisibleTo, shouldn't the compiler/runtime already apply any benefits of sealing to non-extended internal types?"  But maybe that's asking for too much magic in the compiler.